Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -27,7 +27,7 @@ private struct BrowserViewControllerUX {
 
 class BrowserViewController: UIViewController {
 
-    private var urlBar: URLBarView!
+    var urlBar: URLBarView!
```

Comment:
That you've had to shed the `private` nature of this is a sign you're doing something wrong. Don't break encapsulation.